### PR TITLE
OPS: Device Registration Lane

### PR DESCRIPTION
**Pull request by marcosrdz**
In order to debug on Apple physical devices you gotta register them. This automates the process by providing a txt file

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/mpv5kvmoq4pvbqt7hzvm4puuwu?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/40)

**Review by Overtorment — APPROVED** [review]
@marcosrdz a few words wtf this is would be nice

File: ios/fastlane/Fastfile [hunk 1]
```diff
@@ -7,6 +7,32 @@ platform :ios do
     sh("git checkout -- .")
   end
 
+  desc "Register new devices from a file"
+  lane :register_devices_from_txt do
+    UI.message("Registering new devices from file...")
+
+    csv_path = "../../devices.txt" # Update this with the actual path to your file
```
Comment (Overtorment) [inline ios/fastlane/Fastfile:14]: What's supposed to be in this file? Do we need to put it in PR as well?

**Review by Overtorment — APPROVED** [review]

**Comment by marcosrdz** [comment]
> @marcosrdz a few words wtf this is would be nice

Done

File: ios/fastlane/Fastfile (hunk 1 above)
Comment (marcosrdz) [reply ios/fastlane/Fastfile:14]: No. It's a standard Apple multi device txt file. Device identification is in it so it's something that shouldn't be committed.

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/obxhshvjlcxenqjhbnazwhjrum?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/41)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**